pkg/declcfg/declcfg.go

```diff
@@ -0,0 +1,82 @@
+package declcfg
+
+import (
+	"encoding/json"
+
+	"github.com/operator-framework/operator-registry/pkg/property"
+)
+
+const (
+	schemaPackage = "olm.package"
+	schemaBundle  = "olm.bundle"
+)
+
+type DeclarativeConfig struct {
+	Packages []Package
+	Bundles  []Bundle
+	Others   []Meta
+}
+
+type Package struct {
+	Schema         string `json:"schema"`
```

Comment:
I considered embedding `Meta` in the other blob structs, but decided against it (so far) because:
1. It didn't seem like it added much value.
2. The Meta type has custom marshal/unmarshal functions that I didn't want interfering with these structs. There's probably a way to make this a little prettier, but it didn't seem worth it to spend time on that.